Sources/Responses/String+BytesRepresentable.swift

```diff
@@ -21,4 +22,11 @@ public extension String {
     self.init(statusLine + joinedHeaders)
   }
 
+  init?(bytes: BytesRepresentable) {
+    self.init(bytes: bytes.toBytes, encoding: .utf8)
+  }
+
+  // public static func == (lhs: BytesRepresentable, rhs: String) -> Bool {
+  //   return lhs.toBytes == rhs.toBytes
+  // }
```

Comment:
I know that for XCTest to assert equality, it wants the two things being compared to conform to the `Equatable` protocol, so I tried to extend the `==` operator to String to accept a BytesRepresentable argument, but this did not have any effect.